docs/howto/schema.rst

```diff
@@ -1,16 +1,16 @@
 Set the database schema
 -----------------------
 
-Any PostgreSQL database contains one or more *schemas*. Schemas are PostgreSQL way of
-implementing namespaces for database objects. See the `PostgreSQL Schema documentation`_
-for more detail on schemas. (See the :ref:`glossary <glossary_schema>` for a note on the
-overloading of the term *schema*.)
+Any PostgreSQL database contains one or more schemas. Schemas are
+PostgreSQL way of implementing namespaces for database objects. See the
+`PostgreSQL Schema documentation`_ for more detail on schemas. See the glossary
```

Comment:
```suggestion
`PostgreSQL Schema documentation`_ for more details on schemas. See the glossary
```

Reply:
I guess both are correct :-)
https://english.stackexchange.com/questions/123672/detail-or-details-read-on-for-more-detail-details#127452